sr_robot_lib/test/test_robot_lib.cpp

```diff
@@ -113,6 +113,64 @@ TEST(SrRobotLib, Initialization)
+                                                                                            1.1781, 1.1781, 1.1781,
+                                                                                            1.1781, 1.1781, 1.5708,
+                                                                                            1.5708, 1.5708, 1.5708,
+                                                                                            1.5708, -1.09804, -0.5832,
+                                                                                            0.45440, 1.61843, 2.46427,
+                                                                                            2.66884, 2.154, 1.1164,
+                                                                                            -0.0476325, -0.893474}},
+                                                                                  {"THJ2", {0.6981, 0.34906, 0,
+                                                                                            -0.34906, -0.6981,
+                                                                                            0.6981, 0.34906, 0,
+                                                                                            -0.34906, -0.6981,
+                                                                                            0.6981, 0.34906, 0,
+                                                                                            -0.34906, -0.6981,
+                                                                                            0.6981, 0.34906, 0,
+                                                                                            -0.34906, -0.6981,
+                                                                                            0.6981, 0.34906, 0,
+                                                                                            -0.34906, -0.6981,
+                                                                                            -0.289805, 2.42309,
+                                                                                            4.21045, 4.38956,
+                                                                                            2.892, 0.289805,
+                                                                                            -2.42309, -4.21045,
+                                                                                            -4.38956, -2.892}}};
+
+  for (auto const& x : lib_test->sr_hand_lib->coupled_calibration_map)
+  {
+    for (int i=0; i < x.second.raw_values_coupled_.size(); ++i)
+    {
+      EXPECT_NEAR(x.second.raw_values_coupled_[i], expected_processed_raw_values_coupled[i], 0.01);
+    }
+
+    for (int i=0; i < x.second.calibrated_values_.size(); ++i)
+    {
+      EXPECT_NEAR(x.second.calibrated_values_[i], expected_processed_calibrated_values[x.first][i], 0.01);
+    }
+  }
+}
+
```

Comment:
It would be good to create a test that uses void SrMotorRobotLib<StatusType, CommandType>::calibrate_joint

We could add a test that checks the calibrated value.
And then also a test that calls it 10000 times and checks the time it takes. This would be quite good as we want to make sure it performs well.